objectivec/Tests/GPBWellKnownTypesTest.m

```diff
@@ -99,4 +101,54 @@ - (void)testDuration {
   [duration2 release];
 }
 
+- (void)testAnyPackFromUnpackTo {
+  TestAllTypes *from = [TestAllTypes message];
+  [self setAllFields:from repeatedCount:1];
+  NSData *data = from.data;
+
+  // Test packFrom.
+  GPBAny *any = [[GPBAny alloc] init];
+  [any packFrom:from];
+  XCTAssertEqualObjects(
+      [GPBTypeGoogleApisComPrefix stringByAppendingString:from.descriptor.name],
+      any.typeURL);
+  XCTAssertEqualObjects(data, any.value);
+
+  // Test unpackTo
+  TestAllTypes *to = [TestAllTypes message];
+  BOOL pass = false;
+  [any unpackTo:to];
+  XCTAssertEqualObjects(from, to);
+
+  // Test invalid unpackTo (should fail)
+  ForeignMessage *to2 = [ForeignMessage message];
+  @try {
+    [any unpackTo:to2];
+  }
+  @catch (NSException *exception) {
+    pass = true;
+  }
+  if (!pass) XCTFail("to2's type doesn't match any's type url.");  // this fails. don't understand
+
+  // Test packFrom message of another type.
+  ForeignMessage *from2 = [ForeignMessage message];
+  [any packFrom:from2];
+  XCTAssertEqualObjects(
+      [GPBTypeGoogleApisComPrefix stringByAppendingString:from2.descriptor.name],
+      any.typeURL);
+  // XCTAssertEqualObjects(@"", any.value); // don't know why this failed.
```

Comment:
failed: ((@"") equal to (any.value)) failed: ("") is not equal to ("<>")
Why it's "<>"